pkg/oc/bootstrap/clusteradd/components/router/router_install.go

```diff
@@ -110,7 +111,19 @@ func (c *RouterComponentOptions) Install(dockerClient dockerhelper.Interface, lo
 		return errors.NewError("cannot create aggregate router cert").WithCause(err)
 	}
 
+	// This snowflake makes sure that when the Docker is on remote host, we copy the generated cert into
+	// the Docker host master config dir.
+	if len(os.Getenv("DOCKER_HOST")) > 0 {
+		hostHelper := host.NewHostHelper(c.InstallContext.DockerHelper(), c.InstallContext.ClientImage())
+		remoteMasterConfigDir := path.Join(host.RemoteHostOriginDir, kubeapiserver.KubeAPIServerDirName)
```

Comment:
you should not be copying into this destination.  The add on components don't get to mess with the running kube/openshift control plane.  Copy to the remote host if you must, but this should be a different directory off the basedir.  Perhaps I didn't notice when I was fixing the rest.

Reply:
this was pre-existing (we generated the cert in the master config dir)... I can move this somewhere else.